### ISSUE TITLE
Permissions page's console modding section generates incorrect URL

### ISSUE THREAD

**Issue by DankRafft** [Nexus-Mods/web-issues#1675]
## Your username/username(s) affected
DankRafft

## Bug/error being reported:
I just uploaded one of my Fallout 4 mods to Bethesda.net and wanted to add a mirror link to the mod page on Nexus Mods. I used the Console Modding section in the Permissions tab for that purpose. I put in the file ID to the new page and it generated an URL. But it looks like Bethesda changed their system so that the URLs in their mod database use a different format.
Long story short, the URL generated that way didn't work and instead lead to the Fallout 4 root page of Bethesda.net. I had to manually edit the URL in the "Requirements and mirrors" tab to make the mirror link work.

## Expected result:
New and working URL format: https://mods.bethesda.net/en/fallout4/mod-detail/4336462

## Actual result:
Automatically generated URL: https://mods.bethesda.net/#en/workshop/fallout4/mod-detail/4336462

**Comment by Pickysaurus** [comment]
Can you provide a mod page as an example? It might not be a bug, it could just be the author putting the wrong URL in.
